src/os/CMakeLists.txt

```diff
@@ -71,6 +71,14 @@ endif()
 
 add_library(os STATIC ${libos_srcs} $<TARGET_OBJECTS:kv_objs>)
 
+if(WITH_BLUEFS)
+  add_library(bluefs SHARED 
+    bluestore/BlueRocksEnv.cc)
+  target_include_directories(bluefs PUBLIC ${ROCKSDB_INCLUDE_DIR})
+  target_link_libraries(bluefs global)
+  install(TARGETS bluefs DESTINATION lib)
```

Comment:
how would you use this library w/o header file of `BlueRocksEnv.h`?

Reply:
@liupan1111 ping.

Reply:
@tchaikov sorry for missing your last comment.

I will add it. Thanks.